README.md

```diff
@@ -1,7 +1,236 @@
+<!-- Code generated by "go generate" in github.com/kortschak/gospel; DO NOT EDIT. -->
 # `gospel` [![Build status](https://github.com/kortschak/gospel/workflows/Test/badge.svg)](https://github.com/kortschak/gospel/actions)
 
-The `gospel` program lints Go source code for misspellings in comments and strings. It uses hunspell to identify misspellings and only emits coloured output for visual inspection; don't use it in automated linting.
+The `gospel` program lints Go source files for misspellings in comments,
+strings and embedded files.
+
+It uses hunspell to identify misspellings and makes use of source code
+information to reduce the rate of false positive spelling errors where
+words refer to labels within the source code.
+
 
 ## Installation
 
-Beyond the standard Go installation process, you must also have libhunspell and its header files on your system.
\ No newline at end of file
+Beyond the standard Go installation process, you must also have libhunspell
+and its header files on your system. For a debian-based system this is done
+with `sudo apt install libhunspell-dev`.
+
+
+## Work Flow
+
+`gospel` makes use of module information, and so must be run within a module
+and will only consider source files that are referred to by the module.
+
+`gospel` can be used without configuration in many cases, particularly for
+small source trees where the number of potential errors is likely to be
+small. However, its behaviour can be tuned with [configuration files](#configuration-files).
+The initial setup workflow makes use of the [`.words`](#.words) file to
+build a dictionary of candidate words to ignore.
+
+```
+$ gospel -misspellings=.words ./...
+```
```

Comment:
nit: you can use a tab or four-space indent when the blockquote is just 1 or 2 lines, and then you avoid the extra verbosity of the triple backtick, plus it's nicer to read in plaintext.

Reply:
This is Stockholm syndrome; I find this easier to read.